Anna_Mayilyan/python/25_05/Fibonachi.py

```diff
@@ -0,0 +1,11 @@
+def Fibonacci(n):
+    if n<0:
+        print("Number cant be less than 0")
+    elif n==1:
+        return 0
+    elif n==2:
+        return 1
+    else:
+        return Fibonacci(n-1)+Fibonacci(n-2)
+  
+print(Fibonacci(int(input("Enter number "))))
```

Comment:
- Validation for inputs
![image](https://user-images.githubusercontent.com/80816144/119726044-b44dfa80-be81-11eb-8368-bb4ffbce64b5.png)
- for 0 getting error
![image](https://user-images.githubusercontent.com/80816144/119726071-bd3ecc00-be81-11eb-9bde-bd6ca167c601.png)
- coding style fix with pep8